test/xpu_api/utilities/meta/meta.trans/meta.trans.other/underlying_type.pass.cpp

```diff
@@ -112,11 +113,13 @@ kernel_test1(sycl::queue& deviceQueue)
             static_assert(!has_underlying_type_member<int&&>::value);
             static_assert(!has_underlying_type_member<int*>::value);
             static_assert(!has_underlying_type_member<dpl::nullptr_t>::value);
+#   endif
 #endif
         });
     });
 }
 
+#if !_PSTL_ICC_TEST_UNDERLYING_TYPE_BROKEN
```

Comment:
Let's also merge two checks for entire function: `TEST_STD_VER >= 20 && !_PSTL_ICC_TEST_UNDERLYING_TYPE_BROKEN`

Reply:
Fixed